Sources/PackageModel/Target.swift

```diff
@@ -526,3 +528,80 @@ public final class BinaryTarget: Target {
         try super.init(from: decoder)
     }
 }
+
+public final class ExtensionTarget: Target {
```

Comment:
inheritance :(

Reply:
Right, but that's in line with what the other targets currently do.  Consistency seemed better than to special-case this one target type.